src/components/HorizontalScores.tsx

```diff
@@ -0,0 +1,24 @@
+interface Scores {
```

Comment:
```suggestion
type Scores = {
```
Prefer types over interfaces when reasonable, they are more extensible